### ISSUE TITLE
option to disable CPU mining

### ISSUE THREAD

**Issue by neuronull** [6block/ironfish-gpu-miner#2]
Any plans to add an option to disable CPU mining ?

This would be very useful.

**Comment by username-rand** [comment]
I agree.  Trying to use the miner on my HIVEOS mining rig with a fan cooled intel i3-2310M processor to do GPU mining and my CPU temp gets to 100 degrees C.  That is too hot to run on a continuous basis.  So I cannot use your miner for GPU mining on Nvidia cards until you add an option to not use the CPU for mining.  I have 10 Nvidia cards mostly 30xx that I would like to use to mine IronFish.  Thanks
